### ISSUE TITLE
Lagrange cannot use a non-Gemini search engine

### ISSUE THREAD

**Issue by ElonSatoshi** [skyjake/lagrange#695]
How to reproduce: 

- Go to the Veronica 2 gopherhole: gopher://gopher.floodgap.com:70/1/v2
- Right click on "Search Veronica-2" and click "Use as search URL"
- Try to search for something in the address bar

Expected result: You search for something on Veronica 2

Actual result: Lagrange interprets gopher:// as trying to access the domain name "gopher" through gemini, coming up with a broken URI like this: `gemini://gopher://gopher.floodgap.com:70/7/v2/vs?test`

In other words, it sticks gemini:// in front of gopher:// and fails to load.

**Comment by skyjake** [comment]
Fixed for the upcoming 1.18.4.

**Comment by ElonSatoshi** [comment]
Until the next release, there is a sort of workaround I found:

Add snippets for non-Gemini search engines. For example, I added veronica2 under !v2. Now I can just type "!v2 chili recipe" to search gopher, for example.
